docs/content/ecs/logging.md

```diff
@@ -8,8 +8,9 @@ The SpatialOS GDK for Unity uses a custom `ILogDispatcher` interface instead of
 
 *  `LoggingDispatcher`, which simply logs to the Unity console
 *  `ForwardingDispatcher`, which logs to the Unity console and sends it to the SpatialOS Console
+    * Note: By default, messages with log level `LogType.Log` are not send to SpatialOS. This can be changed by instantiating the `ForwardingDispatcher` with a different `minimumLogLevel` parameter.
 
-All workers use the `ForwardingDispatcher` by default. To replace it with the `LoggingDispatcher`, see the last step of [Creating and using your own dispatcher](#creating-and-using-your-own-dispatcher).
+All workers use the `ForwardingDispatcher` by default in the Playground. If you want to use the `LoggingDispatcher`, see the last step of [Creating and using your own dispatcher](#creating-and-using-your-own-dispatcher).
```

Comment:
Nitpicking: I am wondering if we should mention the `LoggingDispatcher` at all. I was thinking something along the line of "If you would like to customize logging further, see \[creating and using your own dispatcher\]"

Reply:
Given that it exists I think we should mention it. If we don't want it at all then we should remove it from here and the docs.